app/errors.py

```diff
@@ -0,0 +1,14 @@
+class NotWearingMaskError(BaseException):
+    pass
+
+
+class VaccineError(Exception):
```

Comment:
Vaccine error should be initialized (use init)

Example:
class VaccineError(Exception):
def init(self, message):
super.init(message)
self.message = message

Also you can add str to return a default message:
def str(self):
return self.message